index.html

```diff
@@ -1,15 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Hello Microverse</title>
+    <title>Portfolio</title>
 </head>
+
 <body>
-    <header>
-        <h1>Hello Microverse</h1>
+    <header class="header_container">
+        <div class="item header_img"></div>
+        <div class="item menu_icon"></div>
     </header>
+    <main>
+        <div class="container">
+            <div class="item-1">
+                <h1 class="text_title item-1">Vanessa Bonito</h1>
+            </div>
+            <div class="item-2">
+                <p class="text_style">
+                    Hello! I am a software developer! I can help you build a product,
+                    feature or website. Take a look of my works.
+                    If you like what you see and have a project you need coded,
+                    don’t hesitate and contact me.
+                </p>
+            </div>
+            <div class="item-3">
+                <button type="button" class="dark_button">Star collaboration</button>
```

Comment:
- [ ] Kindly wrap the `<button>` tag with a `<a>` tag to ensure proper use of HMTL semantics

Reply:
Hi, @rloterh 

I've already wrapped the <button> tag with a <a> tag, but it gives me a web hint error:
The element “button” must not appear as a descendant of the “a” element.  html-checker

Any suggestions?

Thanks.

Reply:
Hello @VaneCode
Kindly use a `<div>` tag instead of button `<tag>`, or simply remove the `<button>` tag.

Cheers! 😃

Reply:
Hello @rloterh 

Please, can you explain why the use of the tag <button> isn't correct?

Thanks.

Reply:
Hello @VaneCode 

Kindly reference this [FCC link](https://www.freecodecamp.org/news/html-button-link-code-examples-how-to-make-html-hyperlinks-using-the-href-attribute-on-tags/), it clearly explains the concept further.

Cheers!

Reply:
- [ ] This issue is well resolved :heavy_check_mark: